pkg/scalers/nats_jetstream_scaler.go

```diff
@@ -149,11 +152,24 @@ func parseNATSJetStreamMetadata(config *ScalerConfig) (natsJetStreamMetadata, er
 	}
 
 	meta.scalerIndex = config.ScalerIndex
+
+	meta.useHTTPS = false
+	if val, ok := config.TriggerMetadata["useHTTPS"]; ok {
```

Comment:
This value will be read from a YAML in camel case
```suggestion
	if val, ok := config.TriggerMetadata["useHttps"]; ok {
```